salt/_modules/metalk8s.py

```diff
@@ -218,6 +218,34 @@ def archive_info_from_iso(path):
     return _get_archive_info(result["stdout"])
 
 
+def get_configured_archives():
+    """List the already configured metalk8s archives"""
+    archives = {}
+    base_mount = "/srv/scality"
+    for path in os.listdir(base_mount):
+        if not path.startswith("metalk8s-"):
+            continue
+        path = os.path.join(base_mount, path)
+        archive = archive_info_from_product_txt(path)
+        # Check if the archive is really configured or just mounted
+        env_name = "metalk8s-{0}".format(archive["version"])
+        if not os.path.exists(f"/var/lib/metalk8s/repositories/conf.d/99-{env_name}-registry.inc"):
+            continue
```

Comment:
I'm not sure we want to check this cause if for whatever reason this file does not exist we will not umount the ISO, to me we manage everything in `/srv/scality/` so if there is "something" mounted in this directory that we don't know we just umount it.

Especially since we first reconfigure the registry before actually umount the ISO it means we will never umount ISOs since when we run the state to umount the ISOs the registry file is already deleted